db/import_listener.go

```diff
@@ -42,37 +44,60 @@ func NewImportListener(groupID string) *importListener {
 // Writes DCP stats into the StatKeyImportDcpStats map
 func (il *importListener) StartImportFeed(bucket base.Bucket, dbStats *base.DbStats, dbContext *DatabaseContext) (err error) {
 	il.bucketName = bucket.GetName()
-	il.database = Database{DatabaseContext: dbContext, user: nil}
+	il.dbCtx = dbContext
```

Comment:
It looks like the only thing we're using dbCtx for is dbName and stats (in import_pindex), now that we're not using bucket, and we already have databaseStats (although not import stats) on importListener.  

Does it make sense to just include dbName and the stats references on the importListener struct, and remove dbCtx?  It might minimize future cases of unreleased databaseContexts.